src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -350,10 +353,17 @@ protected WgsMetrics generateWgsMetrics() {
     }
 
     protected long getBasesExcludedBy(final CountingFilter filter) {
+        if(INTERVALS != null){
+            return filter.getFilteredRecords();
+        }
         return filter.getFilteredBases();
```

Comment:
Also, this whole method could be shortened to a single line:
return (INTERVALS != null) ? filter.getFilteredRecords() : filter.getFilteredBases();

Oh, and another thing: protocol in this codebase is to put spaces after the 'if' and before the braces.  Otherwise, you risk incurring the wrath of @nh13.